hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,111 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} qrun -optimize"
+  run_cmd:    "{job_prefix} qrun -simulate"
+
+  build_opts: [ "-timescale 1ns/1ps",
+                "-uvm -uvmhome {QUESTA_HOME}/questasim/verilog_src/uvm-1.2",
+                "-mfcu",
+                "-f {sv_flist}",
+                // List multiple tops for the simulation. Prepend each top level with `-top`.
+                "{eval_cmd} echo {sim_tops} | sed -E 's/(\\S+)/-top \\1/g'",
+                "-voptargs=\"+acc=nr\""
+              ]
+
+  run_opts:   [ "-outdir {build_dir}/sim-vcs/qrun.out",
```

Comment:
The `sim-vcs` is a directory generated by this 3rd-party open source tool called FuseSoC. 
I suggest putting anything related to questa directly in {build_dir}. I wonder if we can set `-outdir` switch in `build_opts` also like this, so that the simulation uses the correct build area:
```
  build_opts: [..., "-outdir {build_dir}", ...]
  run_opts: [..., "-outdir {build_dir}", ...]
```

Reply:
Sure ok, I have set a build director similar to vcs.hjson and xcelium.hjson:
` build_db_dir: "{build_dir}/xcelium.d"`
as 
`build_db_dir: "{build_dir}/sim_questa"`

This is now in `build_opts` and `run_opts`

Reply:
Actually just realised that I don't need the extra sim_questa folder. I can do it directly like you said.